auth_otp/views_extended.py

```
@@ -0,0 +1,310 @@
+            },
+            status=status.HTTP_500_INTERNAL_SERVER_ERROR
+        )
+
+
+# ====================================
+# Register Endpoint (Optional)
+# ====================================
+
+@api_view(['POST'])
+@permission_classes([AllowAny])
+def register(request):
```

Ruff (0.12.2)

Missing return type annotation for public function `register`

(ANN201)